Main/Source/rooms.cpp

```diff
@@ -700,7 +700,7 @@ truth bananadroparea::PickupItem(character* Hungry, item* Item, int)
 
   if(Hungry->IsPlayer())
   {
-    if(!Item->IsBanana() && !Item->IsLanternOnWall())
```

Comment:
Did you drop the holy banana there by mistake?

Reply:
No no, bananadroparea is used for Decos' house, so the old code meant that players could take anything inside except for bananas and lanterns without repecussions. Thus they could steal without angering NA.

I changed it to only allowing free taking of quest items (the player would have to bring them there and mistakenly drop them), so players can no longer steal from Decos.